lib/mayaUsd/render/vp2RenderDelegate/meshViewportCompute.cpp

```diff
@@ -1324,11 +1330,15 @@ bool MeshViewportCompute::canConsolidate(const MPxViewportComputeItem& other) co
     // If the compute has executed then the data to be consolidated will already
     // be smoothed. Smoothed items can only consolidate with other smoothed items.
     return hasExecuted() == otherMeshViewportCompute->hasExecuted()
+        && _meshSharedData->_renderingTopology.GetScheme()
```

Comment:
Only allow items that share the same values for scheme, orientation and refine level to consolidate together.